pkg/codegen/schema.go

```diff
@@ -401,8 +401,21 @@ func GenerateGoSchema(sref *openapi3.SchemaRef, path []string) (Schema, error) {
 		for i, enumValue := range schema.Enum {
 			enumValues[i] = fmt.Sprintf("%v", enumValue)
 		}
+		if 0 < len(schema.ExtensionProps.Extensions) {
+			//fmt.Fprintf(os.Stderr, "%#v\n\n", schema)
+		}
```

Comment:
Remove old debugging?
```suggestion
		if 0 < len(schema.ExtensionProps.Extensions) {
			//fmt.Fprintf(os.Stderr, "%#v\n\n", schema)
		}
```